backend/app/services/chatbot_service.py

```
@@ -109,15 +111,15 @@ async def read_chatbot(self, user_id: ObjectId, chatbot_id: str):
             if not existing_chatbot:
                 return JSONResponse(
                     status_code=HTTPStatus.NOT_FOUND,
-                    content={"message": f"Chatbot not found"},
+                    content={"message": fCHATBOT_NOT_FOUND_MSG},
```

Ruff (0.8.2)

Undefined name `fCHATBOT_NOT_FOUND_MSG`

(F821)